app/src/main/java/io/gnosis/safe/ui/base/BaseBottomSheetDialogFragment.kt

```diff
@@ -0,0 +1,49 @@
+package io.gnosis.safe.ui.base
+
+import android.app.Dialog
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.viewbinding.ViewBinding
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import io.gnosis.safe.HeimdallApplication
+import io.gnosis.safe.R
+import io.gnosis.safe.di.components.ApplicationComponent
+
+abstract class BaseBottomSheetDialogFragment<T : ViewBinding> : BottomSheetDialogFragment() {
+
+    private var _binding: T? = null
```

Comment:
why do we start a variable name with `_` isn't that super uncommon for our style guidelines? (shouldn't we use something like `internalBinding` or try to refactor it so that we don't need this dirty hack)

Reply:
It's common in kotlin to use underscore for backing properties. See https://kotlinlang.org/docs/reference/coding-conventions.html